router/transformer/transformer.go

```
@@ -209,62 +221,45 @@ func (trans *HandleT) ResponseTransform(ctx context.Context, responseData integr
 		if resp != nil && resp.Body != nil {
 			tempRespData, _ = io.ReadAll(resp.Body)
```

Not introduced in this PR, but the error is ignored here.